src/azimuth/gui/screen.c

```diff
@@ -52,6 +51,19 @@ static int current_screen_height = AZ_SCREEN_HEIGHT;
 static float current_screen_scale = 1.0f;
 static float current_screen_xoffset = 0;
 static float current_screen_yoffset = 0;
+static double NANOSECONDS_PER_COUNT = 1000000000;
+
+static uint64_t az_current_time_nanos(void) {
```

Comment:
Could you copy over the doc comments for these functions (`az_current_time_nanos` and `az_sleep_until`) from the deleted `timer.h` file (adjusted as necessary), so we don't lose those?

(One detail to adjust in the doc comment is that it doesn't look like [`SDL_GetPerformanceCounter`](https://wiki.libsdl.org/SDL_GetPerformanceCounter) actually guarantees monotonicity, although I checked the implementation and it looks like it tries to use a monotonic clock when available.)